Issues with `make run-cdb-migrations`

While attempting to run `make run-cdb-migrations` in Chapter 5 (pg 189) for the cockroach database, I get the following error:

```
[go get] installing golang-migrate cmd with cockroachdb support
go get github.com/golang-migrate/migrate/cmd/migrate: module github.com/golang-migrate/migrate@upgrade found (v3.5.4+incompatible), but does not contain package github.com/golang-migrate/migrate/cmd/migrate
make: *** [migrate-check-deps] Error 1
```

Any suggestions?

--- a/Makefile
+++ b/Makefile
@@ -40,7 +40,7 @@ run-cdb-migrations: migrate-check-deps check-cdb-env
 migrate-check-deps:
 	@if [ -z `which migrate` ]; then \
 		echo "[go get] installing golang-migrate cmd with cockroachdb support";\
-		go get -tags 'cockroachdb postgres' -u github.com/golang-migrate/migrate/cmd/migrate;\
+		go get -tags 'cockroachdb postgres' -u github.com/golang-migrate/migrate/v4/cmd/migrate;\
 	fi
 
 

